src/dos/dos.cpp

```diff
@@ -1204,6 +1204,25 @@ static Bitu DOS_26Handler(void) {
     return CBRET_NONE;
 }
 
+void set_ver(char *args, bool start)
```

Comment:
`char *args` implies `args` is being modified; can it become `const char* args`?
(I see that StripWord require a modifiable `char*`).

Reply:
Also, `bool start` is not needed and should be removed.

Reply:
Done ("const char*" instead of "char *" and removed "bool start").